packages/babel-plugin-transform-runtime/src/index.js

```diff
@@ -254,7 +254,14 @@ export default declare((api, options, dirname) => {
         if (cached) {
           cached = t.cloneNode(cached);
         } else {
-          cached = addDefault(file.path, source, {
+          const moduleTransforms = file.get(
+            "@babel/plugin-transform-modules-*",
```

Comment:
IIRC, this was only supported since 7.4.0.
Let's try anyway if it is enough. Otherwise, we will have to revert the whole PR which introduced the `.js` extensions and only enable it behind an option.